examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -54,3 +54,10 @@
                        cut_coords=range(0, 51, 10), title='Slices')
 
 plotting.show()
+
+
+##############################################################################
+# Note: `fetch_surf_fsaverage` takes a "mesh" argument which specifies
+# wether to fetch the low-resolution fsaverage5 mesh, or the high-resolution
+# fsaverage mesh. using mesh="fsaverage" will result in more memory usage and
+# computation time, but finer visualizations.
```

Comment:
Here it would be a good thing to show the same plot with the high resolution mesh.